### ISSUE TITLE
fully connected layers

### ISSUE THREAD

**Issue by victorzhucompass** [kentsyx/Neural-IMage-Assessment#20]
This is great work. Thanks. Just wondering are there any particular reason the model does not include fully connected layers like VGG16 before softmax.

**Comment by kentsyx** [comment]
I actually missed that long time ago, by adding 
```
fc1 = list(model.classifier.children())[0]
```
to ```model.py``` should do the trick

**Comment by victorzhucompass** [comment]
great, any update after adding this layer into the model and retrain? wondering whether the learning rates are same setup.
